src/ImageSharp/Processing/Processors/Normalization/HistogramEqualizationOptions.cs

```diff
@@ -32,14 +32,13 @@ public class HistogramEqualizationOptions
         public bool ClipHistogram { get; set; } = false;
 
         /// <summary>
-        /// Gets or sets the histogram clip limit in percent of the total pixels in a tile. Histogram bins which exceed this limit, will be capped at this value.
-        /// Defaults to 0.035f.
+        /// Gets or sets the histogram clip limit. Histogram bins which exceed this limit, will be capped at this value. Defaults to 350.
```

Comment:
What is the range allowed?

Reply:
I would have assumed it was connected to the luminance levels which with the percentile value could be applied to both 8 and 16 bit images. It can't now so we should have some sort of range validation to ensure we're doing the right thing.

Reply:
This depends on the number of pixels in a tile. So its depended on the number of tiles choosen and the image size itself. For example if the image is 1000x1000 pixels and the tiles choosen is 10, the image would be split up into 10 by 10 tiles with each 100 x 100  pixels. In the worst case all pixels in a tile would have the same color and land in the same histogram bin. So a clipLimit of 100x100 pixels would be the upper limit.

There is a Guard in place in the HistogramEqualization constructor ensuring its at least 1: [ClipLimit Guard](https://github.com/brianpopow/ImageSharp/blob/67ea7b43c2e7027fc020a655be6d39c00b27ace9/src/ImageSharp/Processing/Processors/Normalization/HistogramEqualizationProcessor%7BTPixel%7D.cs#L41)

I am not sure what a reasonable upper limit could be. I think this has to be left to the user to choose. The worst which can happen is that the user chooses a clipLimit which is higher then actually possible. In the above example that would be a value higher than 10000. In this case it would just mean the histogram would not be clipped at all.

Reply:
Is it equivalent to the OpenCV implementation now? Does it make sense to reference [their docs](https://docs.opencv.org/3.4/d5/daf/tutorial_py_histogram_equalization.html) from ours as  clarification?

Reply:
If not exactly equivalent we should at least expand upon the documentation for the `clipLimit` property to make it more clear. Once we do that I am happy to merge the code. 👍

Reply:
I have tried to clarify the meaning of the clipLimit in the comment and linked to [wikipedia AHE](https://en.wikipedia.org/wiki/Adaptive_histogram_equalization#Contrast_Limited_AHE)

Reply:
Perfect!